packages/cloudflare/package.json

```diff
@@ -51,7 +51,8 @@
     "fast-glob": "^3.3.1",
     "@types/iarna__toml": "^2.0.2",
     "strip-ansi": "^7.1.0",
-    "astro": "^4.2.0",
+    "astro": "^4.3.5",
+    "chai": "^4.3.10",
```

Comment:
I think this is a mistake

Reply:
This came up in the conflict. I thought it was recently added to `main`. Was it removed?